src/20_Components/amp-bodymovin-animation.html

```diff
@@ -0,0 +1,55 @@
+<!-- 
+  ## Introduction
+
+  Embed Bodymovin animations in your AMP HTML files.
```

Comment:
Can you add a link to the official amphtml doc for the component .

Reply:
Done. I also added a link to the supported actions.